soccer/src/soccer/strategy/agent/position/seeker.cpp

```diff
@@ -37,7 +37,7 @@ rj_geometry::Point Seeker::calculate_open_point(double current_prec, double min_
     while (current_prec > min_prec) {
         rj_geometry::Point ball_pos = world_state->ball.position;
         rj_geometry::Point min = current_point;
-        double min_val = Seeker::eval_point(ball_pos, current_point, world_state);
+        double min_val = Seeker::eval_point(ball_pos, current_point, world_state, field_dimensions);
```

Comment:
Would this also work? just slightly easier to read this way
```suggestion
        double min_val = eval_point(ball_pos, current_point, world_state, field_dimensions);
```